CITATION.cff

```diff
@@ -194,5 +194,5 @@ license: "Apache-2.0"
 message: "If you use this software, please cite it using these metadata."
 repository-code: "https://github.com/ESMValGroup/ESMValCore/"
 title: ESMValCore
-version: "v2.6.0rc2"
+version: "v2.6.0rc3"
```

Comment:
Please make sure to also change `date-released` (GitHub won't let me do that here)